### PR TITLE
Lambda file package modifica v3

### PR DESCRIPTION
**Pull request by matikormos**
Regarding comments of merge request #3, from @martincastrocm, the following changes were made:

- Dynamic enviroments inside the module. 
- Comments deleteds.
- Have been created a map(string) as policy arn variable, in order to accepted many arn policies. 
- Readme updated.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -65,4 +65,9 @@ variable "iam_source_name" {
   default = {
     default = "*"
   }
+}
+
+variable "lambda_policy_arn" {
+  description = "The ARNs of the policies to attach to the lambda role"
+  type        = map(string)
```
Comment (martincastrocm) [inline variables.tf:72]: `default = {}`

File: examples/hello_world/policies/Watchlog.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (martincastrocm) [inline examples/hello_world/policies/Watchlog.json]: move this one to the module and attached it as "inline policy": if you're creating always the log group you should have logging capabilities. You can use [iam_role_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_role_policy)

File: variables.tf (hunk 1 above)
Comment (matikormos) [reply variables.tf:72]: Done, commit: 79e714e4a1477d97d31c1a916d9f2c09e3bd527d

File: examples/hello_world/policies/Watchlog.json (hunk 2 above)
Comment (matikormos) [reply examples/hello_world/policies/Watchlog.json]: Done, commit: 79e714e4a1477d97d31c1a916d9f2c09e3bd527d

File: lambda.tf [hunk 3]
```diff
@@ -49,22 +44,25 @@ resource "aws_iam_role" "lambda_role" {
...
+  for_each = var.lambda_policy_arn
 
-data "template_file" "example" {
-  template = file("${path.module}/policies/Watchlog.json.tpl")
-
-  vars = {
-    resource = "${var.iam_source_name.default}"
-  }
+  role       = aws_iam_role.lambda_role.name
+  policy_arn = var.lambda_policy_arn[each.key] #element(var.lambda_policy_arn, count.index)
```
Comment (martincastrocm) [inline lambda.tf]: `each.value` = `var.lambda_policy_arn[each.key]` by definition 
delete the comment

**Review by martincastrocm — APPROVED** [review]
LGTM

File: lambda.tf [hunk 4]
```diff
@@ -49,22 +44,25 @@ resource "aws_iam_role" "lambda_role" {
...
-  policy = data.template_file.example.rendered
+data "aws_iam_policy_document" "policy_lambda_logs" {
+  statement {
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+    effect    = "Allow"
+    resources = ["arn:aws:logs:*:*:*"]
```
Comment (martincastrocm) [inline lambda.tf]: ` resources = [aws_cloudwatch_log_group.lambda_cwgroup.arn]`
test this to ensure the lambda is still able to write logs

**Review by martincastrocm — APPROVED** [review]

File: lambda.tf (hunk 4 above)
Comment (matikormos) [reply lambda.tf]: Done, commit: 8f78c78f9c18aca02ee307e6c070f7c8905196e4

File: lambda.tf (hunk 3 above)
Comment (matikormos) [reply lambda.tf]: Done, commit: 8f78c78f9c18aca02ee307e6c070f7c8905196e4

**Pull request merged**